### PR TITLE
Fixes: #234 - Added test cases

### PR DESCRIPTION
**Pull request by isuruAb**
Fixes issue #234

Changes:Added Test case to each and every component.

### CODE REVIEW

**Comment by saurabhjn76** [comment]
@isuruAb Please check for Travis build

File: src/setupTest.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+window.matchMedia = window.matchMedia || function() {
```
Comment (rishiraj824) [inline src/setupTest.js]: @isuruAb Do we require this?

Comment (uday96) [inline src/setupTest.js]: Yes, I dont think its needed here, webchat needed this to support react-slick

Comment (isuruAb) [inline src/setupTest.js]: window.matchMedia  is used in SkillRollBack.js
so we have to use this.

**Comment by isuruAb** [comment]
@saurabhjn76  @rishiraj824  @uday96  @madhavrathi  please review this

**Pull request merged**